### DISCUSSION TITLE
How to know if a font is embedded font or system font?

### DISCUSSION THREAD

**Discussion by moliqingwa** [pymupdf/PyMuPDF#2604] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
```
doc = fitz.open(filename=file_path, stream=stream, filetype='pdf')

fonts = []
for page_no in range(doc.page_count):
    font_ans = doc.get_page_fonts(page_no, full=True)
    # we need a way to check the font is embedded or system font ???
```

**Describe the solution you'd like**

**Describe alternatives you've considered**
Are there several options for how your request could be met?

**Additional context**

**Comment by JorjMcKie** [comment, marked as answer]
PyMuPDF does not support using system fonts.
A font is either embedded or a Base14 font. This difference is visible by looking at `page.get_fonts()` - as you did.
If you see a "n/a" then the font is not embedded, otherwise it is.
If not embedded then it is a standard font like a Base-14.
It **_never_** is a system font.

**Reply by moliqingwa** [reply]
Got it, thanks :)
